src/assets/ArrowDownIcon.tsx

```diff
@@ -0,0 +1,28 @@
+export default function ArrowDownIcon() {
```

Comment:
there should be a way to use svgs as files without needing components? (see alex's branch)

Reply:
I tried this for like at least 30 minutes and no matter what I did I could not get it to work. Also Alex didn't even use components in his branch, he used img and then used it as a source.